aircraft/C130/C130.xml

```diff
@@ -128,10 +134,11 @@
         </engine>
         <engine file="t56">
             <feed>1</feed>
+            <feed>4</feed>
             <thruster file="t56_prop">
                 <location unit="IN">
                     <x> 586.5 </x>
-                    <y> -265 </y>
+                    <y> 265 </y>
```

Comment:
With this change 3 engines will be located on the right hand side and only one on the left hand side. I am quite skeptical that this change is meaningful.

Granted however that the current version of c130 is erroneous since both right engines are located at the same `Y` coordinate which should not be the case (the same applies to the left engines except the sign of their `Y` coordinates are opposite).